src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -778,6 +795,8 @@ void VtolAttitudeControl::task_main()
 
 				// scale pitch control with total airspeed
 				scale_mc_output();
+				// limit minimum thrust to prevent lack of airflow over control surfaces in hover
+				//limit_min_thrust();
```

Comment:
Yeah, we can close this PR, I was using it when I flew the VTOL in combination with the mc_pos_controller for attitude control. But now there is a vtol_pos_control module which takes care of all of this. It is much cleaner anyway.